src/locales/cn/translation.json

```diff
@@ -204,7 +204,7 @@
     "faqQuestion3": "How will the light fixtures and lanterns be installed for “Light Up Chinatown”?",
     "faqAnswer3": "<0>Alternative Earth Care</0> has a four man team that will install 14 sections of lighting on Mott Street from Canal to Bayard with lanterns hung between each light bulb for Light Up Chinatown.",
     "faqQuestion4": "I donated $150+, where can I pick up my personalized lantern?",
-    "faqAnswer4": "Pick up location is at CCBA, 62 Mott Street, New York, NY 10013. You will receive an email confirmation with detailed instructions on what time to pick up the lantern.",
+    "faqAnswer4": "The pick up location for lanterns will be at Pearl River Mart on 395 Broadway, New York, NY 10013 after 12/26 anytime from 11am - 6pm. You should have received an email confirmation with detailed instructions on pick up. If you provided  your shipping address, we will ship you your personalized lantern after December 21st.",
```

Comment:
There is an extra white space in “you should have provided your shipping address...”

Reply:
Fixed!